dragonchain/webserver/lib/dragonnet.py

```diff
@@ -80,13 +87,48 @@ def process_receipt_v1(block_dto: Dict[str, Any]) -> None:
                 _log.exception("matchmaking add_receipt failed!")
             # Update the broadcast system about this receipt
             broadcast_functions.set_receieved_verification_for_block_from_chain_sync(l1_block_id, level_received_from, block_model.dc_id)
+            # Send verifications to any configured urls
+            send_verification_notifications(level_received_from, block_model)
         else:
             _log.warning(
                 f"Chain {block_model.dc_id} (level {level_received_from}) returned a receipt that wasn't expected (possibly expired?) for block {l1_block_id}. Rejecting receipt"  # noqa: B950
             )
             raise exceptions.NotAcceptingVerifications(f"Not accepting verifications for block {l1_block_id} from {block_model.dc_id}")
 
 
+def get_notification_urls(key: str) -> set:
+    try:
+        urls = set(VERIFICATION_NOTIFICATION[key])
+    except KeyError:
+        urls = set()
+    return urls
+
+
+def sign(message: str) -> Tuple[str, Optional[str]]:
+    dc_keys = keys.DCKeys()
+    signature = dc_keys.make_signature(message.encode("utf8"), crypto.SupportedHashes.sha256)
+    return (signature, dc_keys.pub)
+
+
+def send_verification_notifications(level: int, block_model: "model.BlockModel"):
+    try:
+        if VERIFICATION_NOTIFICATION:
+            string_json = json.dumps(block_model.export_as_at_rest())
+            (signature, public_key) = sign(string_json)
+            all_endpoints = get_notification_urls("all").union(get_notification_urls(f"l{level}"))
+            for url in all_endpoints:
+                _log.debug(f"Notify -> POST {url} {string_json}")
+                response = requests.post(
+                    url,
+                    string_json,
+                    headers={"Content-Type": "application/json", "Authorization": f"Bearer {signature}", "DragonchainId": public_key},
```

Comment:
I'm confused as this isn't really how bearer tokens work. If you want to pass the signature in the header, I would probably just use a custom header for less confusion